manifests/cf-manifest/manifest/800-logsearch.yml

```diff
@@ -40,6 +40,7 @@ jobs:
   - name: cf
     static_ips:
       - 10.0.16.14
+      - 10.0.16.15
```

Comment:
How does this work in dev where we have just 1 instance, will it just ignore the extra static IP? \cc @alext @LeePorte

Reply:
Apparently not... We're going to revert these changes.